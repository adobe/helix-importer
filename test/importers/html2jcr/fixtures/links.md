+-------------------------------------------------------------------------+
| block-links                                                             |
+=========================================================================+
| [Adobe](https://www.adobe.com)                                          |
+-------------------------------------------------------------------------+
| [Google](https://www.google.com)                                        |
+-------------------------------------------------------------------------+