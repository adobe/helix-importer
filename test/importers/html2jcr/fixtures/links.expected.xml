<?xml version="1.0" encoding="UTF-8"?>
<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" jcr:primaryType="cq:Page">
    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" sling:resourceType="core/franklin/components/page/v1/page">
        <root jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/root/v1/root">
            <section sling:resourceType="core/franklin/components/section/v1/section" jcr:primaryType="nt:unstructured">
                <block sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="block-links" filter="block-links" model="block-links">
                    <item jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/block/v1/block/item" name="block-link" link="https://www.adobe.com" linkText="Adobe" model="block-link"/>
                    <item_0 jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/block/v1/block/item" name="block-link" link="https://www.google.com" linkText="Google" model="block-link"/>
                </block>
            </section>
        </root>
    </jcr:content>
</jcr:root>